### PR TITLE
Fixed #30975 -- Replaced custom get_select_option with Selenium's select_by_value.

### PR DESCRIPTION
**Pull request by codingjoe**
https://code.djangoproject.com/ticket/30975

### CODE REVIEW

**Comment by codingjoe** [comment]
buildbot, test on selenium.

**Comment by codingjoe** [comment]
buildbot, test on selenium.

File: django/contrib/admin/tests.py [hunk 1]
```diff
@@ -133,17 +133,15 @@ def get_css_value(self, selector, attribute):
         return self.selenium.execute_script(
             'return django.jQuery("%s").css("%s")' % (selector, attribute))
 
-    def get_select_option(self, selector, value):
+    def select_option(self, selector, value):
         """
-        Return the <OPTION> with the value `value` inside the <SELECT> widget
-        identified by the CSS selector `selector`.
+        Select the <OPTION> with the value `value` inside the <SELECT> widget
+        identified by the CSS selector `selector.
```
Comment (pauloxnet) [inline django/contrib/admin/tests.py]: You missed a backtick at the end of the line.

Comment (codingjoe) [reply django/contrib/admin/tests.py]: Good catch, thanks!

**Comment by codingjoe** [comment]
buildbot, test on selenium.

**Comment by codingjoe** [comment]
buildbot, test on selenium.

File: django/contrib/admin/tests.py (hunk 1 above)
Comment (pauloxnet) [reply django/contrib/admin/tests.py]: You're welcome!

File: tests/admin_widgets/tests.py [hunk 2]
```diff
@@ -1093,30 +1097,30 @@ def execute_basic_operations(self, mode, field_name):
...
         # Choose some more options --------------------------------------------
-        self.get_select_option(to_box, str(self.jason.id)).click()
-        self.get_select_option(to_box, str(self.john.id)).click()
+        self.select_option(to_box, str(self.jason.id))
+        self.select_option(to_box, str(self.john.id))
 
         # Confirm they're selected after clicking inactive buttons: ticket #26575
         self.assertSelectedOptions(to_box, [str(self.jason.id), str(self.john.id)])
         self.selenium.find_element_by_id(choose_link).click()
         self.assertSelectedOptions(to_box, [str(self.jason.id), str(self.john.id)])
```
Comment (carltongibson) [inline tests/admin_widgets/tests.py]: The changes here seem to break the assertion on this line. (Ref e00d77c4834b40f06f9bf271da5fdfb526ad8f56)

Comment (codingjoe) [reply tests/admin_widgets/tests.py]: Ah it took me a while. Of course, it only selects. Some clicks have been deselects tho. I added to explicit methods.

**Comment by codingjoe** [comment]
buildbot, test on selenium.

**Comment by carltongibson** [comment]
buildbot, test on selenium

**Pull request merged**